sphinxcontrib_hdl_diagrams/__init__.py

```diff
@@ -372,6 +375,29 @@ def nmigen_to_rtlil(fname, oname):
     cmd = "{python} {script} > {output}".format(python=sys.executable, script=fname, output=oname)
     subprocess.run(cmd, shell=True, check=True)
 
+def vhdl_to_verilog(fname, oname, module, ghdl, yosys):
+    assert os.path.exists(fname)
+
+    if ghdl == "module":
+        yosys_opt = "-m ghdl "
+    elif ghdl == "prebuilt":
+        yosys_opt = ""
+    elif os.path.exists(ghdl):
```

Comment:
I would use `Path(ghdl).exists()`. This is a Python language preference.

Reply:
Are you sure about this? I tried this as an example and got an error:
```python
>>> from os import path
>>> path('conf.py').exists()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'module' object is not callable
```

On a similar note, `os.path` maybe should be replaced with just `path` throughout the code as it's imported independently from os (or the latter should be removed and use `os.path` everywhere). Seems redundant to have both.
```python
import os # line 26
.
.
.
from os import path # line 33
```

Reply:
Sorry. I should have specified:

```py
from pathlib import Path
```

See https://docs.python.org/3/library/pathlib.html, https://docs.python.org/3/library/pathlib.html#pathlib.Path.exists and https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module.